src/pages/Consumer/Create.tsx

```diff
@@ -51,7 +52,7 @@ const Page: React.FC = (props) => {
         .map((item) => item[0]);
       const isValid = Object.keys(plugins).some((name) => authPluginNames.includes(name));
       if (!isValid) {
-        notification.warning({ message: '请启用至少一种身份认证类插件' });
+        notification.warning({ message: formatMessage({ id: 'create.consumer.enable.authentication.plugin' }) });
```

Comment:
The ID starts with create is not a good idea, please use ModuleName.key1.key2 style, e.g ConsumerModule.tip.atLeastOneAuthPlugin